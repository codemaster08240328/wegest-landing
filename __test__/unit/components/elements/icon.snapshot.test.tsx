import renderer from 'react-test-renderer'
import Icon from '@/components/elements/icon'

describe('<Icon> snapshot', () => {
  it('renders correclty', () => {
    expect(renderer.create(<Icon icon='add' />).toJSON()).toMatchSnapshot()
  })

  it('renders with type', () => {
    expect(
      renderer.create(<Icon icon='add' type='outlined' />).toJSON(),
    ).toMatchSnapshot()
  })

  it('renders with a class', () => {
    expect(
      renderer.create(<Icon icon='add' className='class-test' />).toJSON(),
    ).toMatchSnapshot()
  })
})
