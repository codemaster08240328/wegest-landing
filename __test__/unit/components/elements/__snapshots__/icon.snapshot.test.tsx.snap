// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon> snapshot renders correclty 1`] = `
<span
  aria-hidden={true}
  className="material-icons MuiIcon-root"
  data-testid="el-icon"
>
  add
</span>
`;

exports[`<Icon> snapshot renders with a class 1`] = `
<span
  aria-hidden={true}
  className="material-icons MuiIcon-root class-test"
  data-testid="el-icon"
>
  add
</span>
`;

exports[`<Icon> snapshot renders with type 1`] = `
<span
  aria-hidden={true}
  className="material-icons MuiIcon-root material-icons-outlined"
  data-testid="el-icon"
>
  add
</span>
`;
